### ISSUE TITLE
Code lens is not accessible via screen reader

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#15479]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.75.0-insider (Universal) Commit: 5ae522f8563fe48b89470923f77a9732f
- OS Version: macOS Monterey 12.6.2

Steps to Reproduce:

1. Make sure `typescript.referencesCodeLens.enabled` is set to true
2. Enable VoiceOver (System Preferences > Accessibility > VoiceOver)
3. Open a TypeScript file with code lenses, navigate to them
4. Note that they aren't highlighted by the screen reader, nor spoken out loud - I can click on the code lens using my mouse, which will open a side list with the references, and this list will be picked up by the screen reader.
